### ISSUE TITLE
How to select what fastlogin method use?+

### ISSUE THREAD

**Issue by jouniikaheimo** [m1guelpf/laravel-fastlogin#4]
how to choose which option to use (yobikey, fingerprint)
Now i try to use mac Yobikey but everty mac select option fingerprint ?

**Comment by m1guelpf** [comment]
This package currently prefers platform authenticators over global authenticators (like a yubikey), and will only fallback to external keys. Would you want this behavior to be configurable?

**Comment by jouniikaheimo** [comment]
Yes :) that packege is awesome is can choise usb or internal fingerprint
etc.. and how about multiple user same computer different usb key ??

ke 27. tammik. 2021 klo 11.34 Miguel Piedrafita <notifications@github.com>
kirjoitti:
